### ISSUE TITLE
Redirect Request

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#78865]
### Instructions
- Requests must be submitted **AT LEAST** 2 weeks in advance.  Some requests take a significant amount of time and coordination to implement, so start the process as soon as you know you will need one. 
- This issue will be used from initial request through implementation to ensure all individuals working on this are notified of status updates.  Please do not create multiple issues to track differnt steps.
- It is your responsibility to notify VA stakeholders as appropriate.  

### Type of request
- [ ] We are retiring or taking down a page and need to redirect the URL (complete redirect section)
- [ ] We are changing the URL of an existing page (complete redirect section)
- [ ] We need a custom vanity URL (complete vanity URL section)

### Implementation date
When does this request need to be live?
Choose one:
- [ ] On a specific date, connected to a program or publishing requirement. **DATE:**
- [ ] Any day prior to a deadline/latest possible date. **DEADLINE:** 
- [ ] Any day after an earliest possible date. **EARLIEST DATE:**
- [ ] During a specific date range: ***EARLIEST DATE TO LATEST DATE:***
- [ ] No hard timing requirements, publish when possible.

### Redirects 
Current URL  |  Redirect Destination or New URL
---  |  ---
https://www.va.gov/HEALTHBENEFITS/vps/index.asp | https://www.va.gov/resources/how-to-check-in-with-your-smartphone-for-some-va-appointments/

### Vanity URLs 
Vanity URL  |  Landing page
---  |  ---
requested URL | landing page URL

**Link to campaign landing page request issue:**

### Process, Roles and Responsibilities
- [ ] Requesting team: Above information is provided
- [ ] Requesting team: All appropriate VA stakeholders are notified as appropriate
- [ ] IA: Request is vetted and documented and implementation plan is clear 
- [ ] IA: Request is assigned to appropriate team for implementation 
- [ ] Implementation team: Work is complete
- [ ] Implementation team: Validated in production
- [ ] Requesting team: Validates everything is correct in production and closes ticket

**Comment by jennymayoco** [comment]
Hi @FranECross @randimays are we able to pause this work? We're doing the IA review on this and had some follow-up questions for the destination page before vetting the URLs.

**Comment by jennymayoco** [comment]
@RLHecht can I get background on the decision for the destination page? It seems the legacy page is about check-in kiosks while the modernized page is mostly about checking in using a smartphone. Are the check-in kiosks being phased out from medical locations?

**Comment by randimays** [comment]
@jennymayoco Yes, we can hold on merging this until you give us the green light.

**Comment by jennymayoco** [comment]
@randimays thank you! sorry for any confusion, just want to make sure I'm understanding the context of both pages.

**Comment by RLHecht** [comment]
@jennymayoco Correct, the kiosks are being phased out. This page also hasn't been updated since 2016 so it has a lot of outdated and incorrect information. We checked with VHA who said this redirect would be OK. Hope that help! Let me know if you need any more info.

**Comment by randimays** [comment]
@jennymayoco Do you have an idea of timeline for implementing this redirect?

**Comment by jennymayoco** [comment]
@RLHecht thank you for this background! are we able to add content on the new R&S page that let's Veterans know the kiosks are being phased out? Just in case there's still centers with kiosks and if folks are looking for that info or their location doesn't do smartphone check in yet.

@randimays no timeline on this yet! Let's wait to hear back from Randi

**Comment by jennymayoco** [comment]
Hi @RLHecht wanted to check in and see if we are able to edit the R&S page to include info on the kiosks being phased out?

**Comment by jennymayoco** [comment]
Hi @randimays @jilladams @FranECross we won't be making any changes on the destination page at this time so we're good to move forward with this redirect from an IA perspective. I also added a note to capture all casing variations in the current URL. Thanks!

**Comment by FranECross** [comment]
Excellent, @jennymayoco ! I'll poise it for our sprint that starts next week.

**Comment by FranECross** [comment]
@randimays This is no longer blocked. I moved it to Stretch/Next Sprint for inclusion in the sprint starting next week. Also added a note to capture all casing variations in the current URL. Thanks!

**Comment by randimays** [comment]
@FranECross We have the code ready for this; I'll get another review on it and get it merged in!

**Comment by randimays** [comment]
Just opened a new PR for a syntax issue fix. This is not working 100% yet

**Comment by randimays** [comment]
Validated successfully in production. Closing!
